osfoffline/tasks/operations.py

```diff
@@ -215,18 +215,20 @@ def _run(self):
         with self.local.open(mode='rb') as fobj:
             resp = OSFClient().request('PUT', url, data=fobj, params={'name': self.local.name})
         data = resp.json()
+        if resp.status_code == http.client.FORBIDDEN:
+            Notification().error('You cannot sync File: {} as you only have read permission on this project. '.format(self.local.name))
```

Comment:
"Could not sync '{}' in project '{}'. Please verify you have write permission to this project."